Cubical/Categories/Monoidal/Enriched/More.agda

```diff
@@ -0,0 +1,78 @@
+module Cubical.Categories.Monoidal.Enriched.More where
```

Comment:
I know upstream it's `Cubical.Categories.Monoidal.Enriched` but I think all of this should be under `Cubical.Categories.Enriched`